dist/tools/openocd/openocd.sh

```diff
@@ -202,7 +202,7 @@ do_debug() {
             -c 'halt' \
             -l /dev/null" &
     # save PID for terminating the server afterwards
-    OCD_PID=$?
+    OCD_PID=$(($!+2))
```

Comment:
This bugs me too, but I didn't find a better solution using standard POSIX tools. That's why this needs to be tested on many systems. Any suggestions?

Am 12. August 2015 19:17:40 MESZ, schrieb Joakim Gebart <notifications@github.com>:
>> @@ -202,7 +202,7 @@ do_debug() {
>>              -c 'halt' \
>>              -l /dev/null" &
>>      # save PID for terminating the server afterwards
>> -    OCD_PID=$?
>> +    OCD_PID=$(($!+2))
>
>the +2 feels kind of arbitrary. Is it guaranteed to be the same every
>time on every system?
>
>---
>Reply to this email directly or view it on GitHub:
>https://github.com/RIOT-OS/RIOT/pull/3619/files#r36886983